oak_protobuf_idl_build/Cargo.toml

```diff
@@ -6,5 +6,5 @@ license = "Apache-2.0"
 
 [dependencies]
 anyhow = "*"
-prost = "*"
+prost = { version = "*", default-features = false }
```

Comment:
What's the difference between `oak_protobuf_idl` and `oak_protobuf_idl_build` ?

Reply:
the _build version is used in build scripts. It's the same distinction between prost and prost-build, for instance.